src/main/java/com/sap/oss/phosphor/fosstars/advice/oss/OssAdviceContentYamlStorage.java

```diff
@@ -34,8 +34,12 @@
 
   static {
     try {
+      // FIXME (mibo): check if this could be done for all in OssFeatures defined
+      //                features instead of being based on a rating
       DEFAULT = OssAdviceContentYamlStorage.loadFromResource(
-          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssSecurityRating.class));
+          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssArtifactSecurityRating.class));
```

Comment:
Looks like `OssAdviceContentYamlStorage` needs to be take a class as a parameter to load a right resource.

Reply:
See #474